ansible/testbed-cli.sh

```diff
@@ -228,6 +228,11 @@ function add_topo
   # Delete the obsoleted arp entry for the PTF IP
   ip neighbor flush $ptf_ip
 
+  echo "Announcing routes for ${topology}"
+  pushd ../tests
+  pytest test_announce_routes.py --inventory ${vmfile} --host-pattern all --testbed ${topology} --testbed_file ${tbfile} --show-capture no --assert plain --log-cli-level info --skip_sanity --disable_loganalyzer
```

Comment:
```run_test.sh``` will set some environment variable, including ```ANSIBLE_CONFIG```, ```ANSIBLE_LIBRARY``` and ```ANSIBLE_CONNECTION_PLUGINS```. Can we run pytest without these variables?

Reply:
we need to set ANSIBLE_CONFIG for sure. I encountered failure recently when running pytest directly

Reply:
Good catch! Thanks!
My sonic-mgmt used for testing already has these environment variables exported. That's why the issue was not caught in my testing. Indeed I noticed another issue, currently it only works for VS setup. The reason is that the inventory file used for VS setup `veos_vtb` has host information of PTF. For physical test setups, the `veos` file does not have PTF host information, the script would fail. I need to figure out how to make it work for physical test setups as well.

Reply:
Updated this PR with a new approach. Updated PR description.